src/hotspot/cpu/riscv/vm_version_riscv.hpp

```diff
@@ -286,6 +412,7 @@ class VM_Version : public Abstract_VM_Version {
 
   // Null terminated list
   static RVFeatureValue* _feature_list[];
+  static RVExtFeatures* _rv_ext_features;
```

Comment:
Does that need to be a pointer? Can't it be `static RVExtFeatures _rv_ext_features` directly?

Reply:
The reason to put `RVExtFeatures` as a pointer here is that, there is such dependency relationship among classes and macro here: `DECLARE_RV_EXT_FEATURE` -(deps)-> `RVFeatureValue`/`RVExtFeatureValue`/`RVNonExtFeatureValue` -(deps)-> `RVExtFeatures` -> `RVFeatureIndex` -> `DECLARE_RV_EXT_FEATURE`.
One way to break the cyclic deps is to use a pointer of RVExtFeatures in `RVExtFeatureValue`/`RVNonExtFeatureValue`.